xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/transactions/metadata/MetaMpTokenIssuanceObject.java

```diff
@@ -0,0 +1,85 @@
+package org.xrpl.xrpl4j.model.transactions.metadata;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.google.common.primitives.UnsignedInteger;
+import org.immutables.value.Value;
+import org.immutables.value.Value.Immutable;
+import org.xrpl.xrpl4j.model.flags.MpTokenIssuanceFlags;
+import org.xrpl.xrpl4j.model.ledger.ImmutableMpTokenIssuanceObject;
+import org.xrpl.xrpl4j.model.ledger.LedgerObject;
+import org.xrpl.xrpl4j.model.transactions.Address;
+import org.xrpl.xrpl4j.model.transactions.AssetScale;
+import org.xrpl.xrpl4j.model.transactions.Hash256;
+import org.xrpl.xrpl4j.model.transactions.MpTokenObjectAmount;
+import org.xrpl.xrpl4j.model.transactions.TransferFee;
+
+import java.util.Optional;
+
+@Immutable
+@JsonSerialize(as = ImmutableMetaMpTokenIssuanceObject.class)
+@JsonDeserialize(as = ImmutableMetaMpTokenIssuanceObject.class)
+public interface MetaMpTokenIssuanceObject extends MetaLedgerObject {
+
+  @JsonProperty("Flags")
+  Optional<MpTokenIssuanceFlags> flags();
+
+  @JsonProperty("Issuer")
+  Optional<Address> issuer();
+
+  @JsonProperty("AssetScale")
+  Optional<AssetScale> assetScale();
+
+  @JsonProperty("MaximumAmount")
+  Optional<MpTokenObjectAmount> maximumAmount();
```

Comment:
Is there any world where `MpTokenObjectAmount` can be replaced by `MpTokenAmount` (esp if [this suggestion ](https://github.com/XRPLF/xrpl4j/pull/556/files#r1809585860)is taken)?